service/src/test/java/bio/terra/landingzone/library/landingzones/TestArmResourcesFactory.java

```diff
@@ -79,11 +79,11 @@ public static SecurityInsightsManager createSecurityInsightsArmClient() {
   }
 
   public static ResourceGroup createTestResourceGroup(AzureResourceManager azureResourceManager) {
-    String resourceGroupId = UUID.randomUUID().toString();
+    String resourceGroupId = UUID.randomUUID().toString().replace("-", "");
```

Comment:
This was my initial change to fix the error I hit due to WOR-1213. I could revert this, but the MRG name we are using is beyond what we intend to support.